pkg/github/github.go

```diff
@@ -147,3 +151,31 @@ func createIssueLocal(ctx context.Context, ghCreateIssueOptions *CreateIssueOpti
 
 	return existingIssue, nil
 }
+
+// CreateEncryptedSecret creates an encrypted secret using a public key from a GitHub repository, which can be sent through the GitHub API
+// https://github.com/google/go-github/blob/master/example/newreposecretwithxcrypto/main.go
+func CreateEncryptedSecret(secretName, secretValue string, publicKey *github.PublicKey) (*github.EncryptedSecret, error) {
```

Comment:
Could also be extracted to `github/secrets.go` to logically separater the code a bit.

Reply:
Done